packages/components/tooltip/src/tooltip.tsx

```diff
@@ -331,7 +358,14 @@ const Tooltip = (props: TTooltipProps) => {
             })}
             data-placement={popper.placement}
           >
-            <BodyComponent>{props.title}</BodyComponent>
+            <div
+              css={css({
+                ...getTooltipStyles(state),
+              })}
+              data-testid="tooltip-message-wrapper"
```

Comment:
Since `BodyComponent` can be provided by a consumer, we need a wrapper to handle the effect transitions.